src/crypto/CHIPCryptoPALmbedTLS.cpp

```diff
@@ -1407,36 +1408,47 @@ CHIP_ERROR ExtractAKIDFromX509Cert(const ByteSpan & certificate, MutableByteSpan
 
 namespace {
 
-CHIP_ERROR ExtractDNAttributeFromX509Cert(bool isVID, const ByteSpan & certificate, uint16_t & id)
+CHIP_ERROR _ExtractDNAttributeFromX509Cert(const uint8_t * oidAttribute, size_t oidAttributeLen, const ByteSpan & certificate,
```

Comment:
No need for a leading underscore since this is already in Anonymous namespace

Reply:
Fixed in 9cddb31